src/features/bookmarkedPosts/bookmarkedPostsSlice.js

```diff
@@ -36,6 +36,7 @@ export const bookmarkedPostsSlice = createSlice({
         state.status = "succeeded";
         state.data.unshift(action.payload?.post);
         notify("Post added to bookmarks successfully", "success");
```

Comment:
Instead of hard coding notify message you could define an object, it would make easier to edit these messages in future

Reply:
Yes, All notify can be modified from one place.